src/shared/logger/activation.ts

```diff
@@ -101,6 +101,7 @@ export async function activate(
  * @param opts.logPaths Array of paths to output log entries to
  * @param opts.outputChannels Array of output channels to log entries to
  * @param opts.useDebugConsole If true, outputs log entries to currently-active debug console. As per VS Code API, cannot specify a debug console in particular.
+ * @param opts.useBrowserConsole If true, outputs log entries to the Developer Tools console when in the browser.
```

Comment:
This docstring is not really true because of the isInBrowser() check :) That's an example of why such special cases should be avoided: it makes explaining and reasoning about things harder.

Reply:
Updated the code and this string, should be more clear now